src/buffer/out/textBuffer.cpp

```diff
@@ -1869,15 +1869,15 @@ std::string TextBuffer::GenRTF(const TextAndColor& rows, const int fontHeightPoi
 // - lastCharacterViewport - Optional. If the caller knows that the last
 //   nonspace character is in a particular Viewport, the caller can provide this
 //   parameter as an optimization, as opposed to searching the entire buffer.
-// - oldViewportTop - Optional. The caller can provide a row in this parameter
-//   and we'll calculate the position of the _end_ of that row in the new
-//   buffer. The row's new value is placed back into this parameter.
+// - oldRows - Optional. The caller can provide a pair of rows in this parameter

```

Comment:
nit: i'd call it something like "positionInformation" or name the struct that -- it's about resizing position anyway yeah?

Reply:
Yea it's not a great name, I agree